hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -128,69 +125,23 @@ class HassioAddonInfo extends LitElement {
     return html`
       ${this.addon.update_available
         ? html`
-            <ha-card
-              .header="${this.supervisor.localize(
-                "common.update_available",
-                "count",
-                1
-              )}🎉"
+            <ha-alert
```

Comment:
Should this only show on the add-on page for pre 2021.12?

Reply:
Yes